scripts/collect_workloads.py

```
@@ -1015,6 +1098,111 @@ def _pad_to_tokens(msgs: list[dict], approx_tokens: int) -> list[dict]:
+
+        # Collect trace lines from the standard trace output location
+        # flashinfer-bench writes traces to {trace_dir}/traces/<def_name>_workloads.jsonl
+        auto_trace = traces_dir / f"{def_name}_workloads.jsonl"
+
+        failed = []
+        passed = 0
+
+        if auto_trace.exists():
+            import json as _json
+
+            lines = [l.strip() for l in auto_trace.read_text().splitlines() if l.strip()]
```

Ruff (0.15.9)

[error] Ambiguous variable name: `l`

(E741)